internal/home/upgrade_test.go

```diff
@@ -558,6 +560,18 @@ func TestUpgradeSchema12to13(t *testing.T) {
 	}
 }
 
+func TestUpgradeSchema12to13Regression(t *testing.T) {
+	body, err := os.ReadFile("testdata/12to13-regression.yaml")
+	assert.NoError(t, err)
+	diskConf := yobj{}
+	err = yaml.Unmarshal(body, &diskConf)
+	assert.NoError(t, err)
+
+	err = upgradeSchema12to13(diskConf)
+
+	assert.NoError(t, err)
```

Comment:
We usually group `assert`s and `requires` together with the statements where the data was acquired, so please remove the empty line above.